### PR TITLE
Fix CommandBar rendering commandText span when no item name is given:…

### LINKED ISSUE #2233: CommandBar renders commandText span when no item name is given
Fix coming shortly...

### PR DESCRIPTION
**Pull request by trevorsg**
… #2233.

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #2233 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

See title

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx [hunk 1]
```diff
@@ -228,8 +228,15 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
             aria-haspopup={ hasSubmenuItems(item) }
           >
             { (hasIcon) ? this._renderIcon(item) : (null) }
-            <span className={ css(
-              'ms-CommandBarItem-commandText', styles.itemCommandText) } aria-hidden='true' role='presentation'>{ item.name }</span>
+            { (!!item.name) && (
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: is there a reason for !!item.name vs just item.name? Haven't seen us use that notation much recently.

Comment (trevorsg) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: I think it's a silly pattern but I was just following the existing style from Line 191 and 214 :)

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: hah...old patterns begetting old patterns :) The whole component is going to be replaced soon anyway. Just another reminder that I need to work on a best practices example component.

Comment (trevorsg) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: @micahgodbolt If it's worth anything, I _really_ like the [Prettier formatter](https://github.com/prettier/prettier) and the opinions it has on formatting JS/TS/JSX/TSX. It has a VSCode extension that works very well.

**Pull request merged**